plugwise/helper.py

```diff
@@ -581,7 +612,7 @@ def _appliance_types_finder(self, appliance: etree, appl: Munch) -> Munch:
             # Provide a home_location for legacy_anna, preset all types applicable to home
             if self._smile_legacy and self.smile_type == "thermostat":
                 appl.location = self._home_location
-            appl.types = self._loc_data[self._home_location].get("types")
+            # appl.types = self._loc_data["data"].get("types")
```

Comment:
Should we add an indication how/why we commented this?

Reply:
That's a typo, I think, will look at this later.

Reply:
This commented-out line has been corrected.

Strange that commenting out this line doesn't break anything.
On the other hand, we only look for a specific type in line 505. That's probably the reason.